components/StartPage/StartButton/index.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+import { useRouter } from "next/router";
+
+export default function StartButton() {
+  const router = useRouter();
+
+  const onHandleStart = () => {
+    router.push({
+      pathname: "/Quiz",
+    });
+  };
+  return (
+    <StyledStartButton type="button" onClick={onHandleStart}>
+      Start
+    </StyledStartButton>
+  );
+}
```

Comment:
It's way simpler (and better semantics) to use a `next/link` here instead of a button and `router.push`.